pkg/server/commands/write.go

```diff
@@ -12,24 +12,37 @@ import (
 	"github.com/openfga/openfga/pkg/storage"
 	tupleUtils "github.com/openfga/openfga/pkg/tuple"
 	"github.com/openfga/openfga/pkg/typesystem"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 // WriteCommand is used to Write and Delete tuples. Instances may be safely shared by multiple goroutines.
 type WriteCommand struct {
-	logger    logger.Logger
-	datastore storage.OpenFGADatastore
+	logger           logger.Logger
+	datastore        storage.OpenFGADatastore
+	rejectConditions bool
 }
 
 // NewWriteCommand creates a WriteCommand with specified storage.TupleBackend to use for storage.
-func NewWriteCommand(datastore storage.OpenFGADatastore, logger logger.Logger) *WriteCommand {
+func NewWriteCommand(datastore storage.OpenFGADatastore, logger logger.Logger, rejectConditions bool) *WriteCommand {
```

Comment:
I'd recommend using the options builder pattern here for 2 reasons:

1. We're trying to use it elsewhere. I know this is temporary for the `v1.3.8` release, but it's inline with what we're already doing.

2. We can default `rejectConditions` to `false` so we can avoid all of the boilerplate cleanup in tests after the `v1.3.8` release. If the experimental is set then we set it to true, but otherwise it's defaulted to false.